### PR TITLE
Add missing paywalls

### PR DESCRIPTION
**Pull request by mariobalca**
# Changes proposed ✍️
- Add missing paywalls to Eagle Eye and Community Help Center (custom domain)
  
### Screenshots (front-end changes only)
Screenshot 2022-12-23 at 11 13 45
Screenshot 2022-12-23 at 11 13 48
Screenshot 2022-12-23 at 11 14 04

## Checklist ✅
- [x] Label appropriately with `Feature`, `Enhancement`, or `Bug`.
- [x] Tests are passing.
- [ ] New backend functionality has been unit-tested.
- [ ] Environment variables have been updated:
  - [ ] Local frontend configuration: `frontend/.env.dist.local`, `frontend/.env.dist.composed`.
  - [ ] Local backend: `backend/.env.dist.local`, `backend/.env.dist.composed`.
  - [ ] [Configuration docs](https://docs.crowd.dev/docs/configuration) have been updated.
  - [ ] Team members only: update environment variables in override, staging and production env. files and trigger update config script.
- [ ] API documentation has been updated (if necessary) (see [docs on API documentation](https://docs.crowd.dev/docs/updating-api-documentation)).
- [x] [Quality standards](https://github.com/CrowdDotDev/crowd-github-test-public/blob/main/CONTRIBUTING.md#quality-standards) are met.
- [ ] All changes have been tested in a staging site.
- [x] All changes are working locally running crowd.dev's Docker local environment.

### CODE REVIEW

File: frontend/src/modules/layout/components/paywall-modal.vue [hunk 1]
```diff
@@ -70,24 +77,42 @@ const model = computed({
...
-  // TODO: Community Help Center paywall
-  'community-help-center': {}
+  communityHelpCenter: {
+    title: 'Custom domain',
+    preTitle: `Community Help Center`,
+    imageSrc: '/images/paywall/community-help-center.png',
+    imageClass: 'absolute bottom-0 right-0',
+    imageWrapperClass: 'h-30',
+    content:
+      'In order to setup a custom domain to your help center public page its required to upgrade your plan',
```
Comment (joanagmaia) [inline frontend/src/modules/layout/components/paywall-modal.vue:102]: This sentence feels a bit weird. Probably missing a comma before "its", and shouldn't its be it's?

File: frontend/src/modules/layout/pages/paywall-page.vue [hunk 2]
```diff
@@ -88,6 +105,9 @@ const props = defineProps({
 })
 
 const page = computed(() => pageContent[props.module])
+const computedFeaturePlan = computed(() => {
```
Comment (joanagmaia) [inline frontend/src/modules/layout/pages/paywall-page.vue:108]: On this file (line 18) there is also a pre title with "Growth feature". Could you also update that one? 🙏

**Review by joanagmaia — CHANGES_REQUESTED** [review]

File: frontend/src/modules/layout/components/paywall-modal.vue (hunk 1 above)
Comment (mariobalca) [reply frontend/src/modules/layout/components/paywall-modal.vue]: Thanks for noticing this, I blindly copied the copy from Figma 😅

**Review by joanagmaia — APPROVED** [review]
LGTM! 💪

**Pull request merged**